docker-service/templates/docker-compose-service

```diff
@@ -16,6 +16,14 @@
 
 start()
 {
+    STATUS=`/usr/local/bin/docker-compose -p {{service_name}} -f /etc/docker-compose/{{service_name}}/docker-compose.yml ps`
+    if ! echo $STATUS | grep Up; then
```

Comment:
some of the compose services have multiple containers. There can be a state where one or more containers may have exited without affecting the status of the other containers.  This check would miss that.